clik/external/CMakeLists.txt

```diff
@@ -25,6 +25,9 @@ if(CLIK_HAL_DIR STREQUAL "")
   endif()
 endif()
 
+# Clik currently only works with hal cpu in WI mode so disable WG mode
+option(HAL_CPU_WG_MODE "Build Hal CPU in workgroup mode" OFF)
```

Comment:
Beware that this only sets the default, it presents the user with a nice toggle that they can flip which will break things. Do we want to pre-emptively error out here if the user makes that change, or do we just leave it to break later on?

Reply:
I think it's fine if they want to break it, we should assume they've written the extra code to make it work. (although you are right we could set  a message if they do.

Reply:
Sure, that sounds reasonable to me (with or without the message).